test_concatenate.py

```diff
@@ -1,5 +1,6 @@
 from concatenate import concatenate
 
 def test_concatenate():
+    """Test that the concatenate function works as expected."""
```

Comment:
This is interesting, I knew that function shall have docstrings but didn't know that test functions should have it too. Do you maybe have a reference on this matter?

Reply:
You are right, it doesn't say that tests should have docstrings. I thought it makes sense but it doesn't, because the function is already explained, it is obvious that the test is testing the functionality.